lib/EventTimeoutProcessor.ts

```diff
@@ -46,6 +57,7 @@ export class EventScheduler implements IProcessor {
             const queuedMessage = await getQueuedMessage(read, sessionId);
 
             await setIsQueueWindowActive(read, persistence, sessionId, false);
+            await modify.getScheduler().cancelJobByDataQuery({sessionId, jobName: JobName.QUEUE_WINDOW_SCHEDULER});
```

Comment:
We are setting and unsetting `setIsQueueWindowActive` in this function. The actual flag that blocks user messages is `setIsProcessingMessage` So maybe we don't need this here?